microservices/document_manager/backend/app/commands.py

```diff
@@ -0,0 +1,7 @@
+def register_commands(app):
```

Comment:
Can you add a sample test command or something and remove the commented out code?

Reply:
removed commented code.